Mismatched types error in internal/security/kdf/methods.go

# 🐞 Bug Report

```
# github.com/edgexfoundry/edgex-go/internal/security/kdf [github.com/edgexfoundry/edgex-go/internal/security/kdf.test]
./methods.go:84:13: invalid operation: nbytes != saltLength (mismatched types int and int32)
FAIL	github.com/edgexfoundry/edgex-go/internal/security/kdf [build failed]
```

--- a/internal/security/kdf/methods.go
+++ b/internal/security/kdf/methods.go
@@ -28,7 +28,7 @@ import (
 // in the event that the KDF inputKeyMaterial is less than random.
 const (
 	saltFile   string = "kdf-salt.dat"
-	saltLength        = 32
+	saltLength int    = 32
 )
 
 var (

